src/seismometer/controls/cohort_comparison.py

```diff
@@ -21,15 +21,23 @@ def __init__(self, selections: dict[str, tuple[any]], exclude_cols: Optional[lis
 
         for side in ["Left", "Right"]:
             options = selections
-            widget = MultiSelectionListWidget(options=options, title=f"Select {side} Cohort")
+            widget = MultiSelectionListWidget(options=options, title=f"Select {side} Cohort", border=True)
             self.selectors.append(widget)
 
         self.output = Output()
         self.button = Button(description=GENERATE_REPORT, button_style="primary")
         self.button.on_click(partial(self._generate_comparison_report, self))
 
     def show(self):
-        display(VBox(children=[HBox(children=self.selectors), self.button, self.output]))
+        display(
+            VBox(
+                children=[
+                    Box(children=self.selectors, layout=Layout(grid_gap="20px", align_times="flex-start")),
```

Comment:
What is align_times?  
If this is align_items, you could use BOX_GRID_LAYOUT

Reply:
Spelling at its worst, this should be BOX_GRID_LAYOUT. 

FINC